keyboards/monsgeek/m1/m1.c

```diff
@@ -0,0 +1,649 @@
+        { 28, 25}, // 31
+        { 42, 25}, // 32
+        { 56, 25}, // 33
+        { 70, 25}, // 34
+        { 84, 25}, // 35
+        { 98, 25}, // 36
+        {112, 25}, // 37
+        {126, 25}, // 38
+        {140, 25}, // 39
+        {154, 25}, // 40
+        {168, 25}, // 41
+        {182, 25}, // 42
+        {224, 25}, // 43
+
+        {  0, 38}, // 44
+        { 28, 38}, // 45
+        { 42, 38}, // 46
+        { 56, 38}, // 47
+        { 70, 38}, // 48
+        { 84, 38}, // 49
+        { 98, 38}, // 50
+        {112, 38}, // 51
+        {126, 38}, // 52
+        {140, 38}, // 53
+        {154, 38}, // 54
+        {168, 38}, // 55
+        {182, 38}, // 56
+        {196, 38}, // 57
+        {224, 38}, // 57
+
+        {  0, 51}, // 58
+        { 14, 51}, // 59
+        { 28, 51}, // 59
+        { 37, 51}, // 60
+        { 56, 51}, // 61
+        { 74, 51}, // 62
+        { 93, 51}, // 63
+        {112, 51}, // 64
+        {130, 51}, // 65
+        {149, 51}, // 66
+        {168, 51}, // 67
+        {186, 51}, // 68
+        {200, 51}, // 69
+        {214, 51}, // 70
+        {224, 51}, // 71
+
+        {  0, 64}, // 72
+        { 18, 64}, // 73
+        { 37, 64}, // 74
+        { 92, 64}, // 75
+        {140, 64}, // 76
+        {154, 64}, // 77
+        {168, 64}, // 78
+        {196, 64}, // 80
+        {210, 64}, // 81
+        {224, 64},  // 82
+
+        {0,   0}, //  68 LED 1
+        {0,   6}, //  69 LED 2
+        {0,   13}, //  70 LED 3
+        {0,   19}, //  71 LED 4
+        {0,   25}, //  72 LED 5
+        {0,   32}, //  73 LED 6
+        {0,   38}, //  74 LED 7
+        {0,   45}, //  75 LED 8
+        {0,   51}, //  73 LED 9
+        {0,   57}, //  74 LED 10
+        {0,   64}, //  75 LED 11
+
+        {224, 0 }, //  78 LED 12
+        {224, 6 }, //  79 LED 13
+        {224, 13}, //  80 LED 14
+        {224, 19}, //  81 LED 15
+        {224, 25}, //  82 LED 16
+        {224, 32}, //  83 LED 17
+        {224, 38}, //  84 LED 18
+        {224, 45}, //  82 LED 19
+        {224, 51}, //  83 LED 20
+        {224, 57}, //  84 LED 21
+        {224, 64}  //  85 LED 22
+    },
+    {
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,       
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4, 
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4, 
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4, 
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4, 
+		4,  4,  4,          4,              4,  4,  4,  4,  4,  4,
+        2,  2,  2,  2,  2,  2,  2,  2,  2,  2,  2,
+        2,  2,  2,  2,  2,  2,  2,  2,  2,  2,  2      
+    }
+};
+
+// clang-format on
+bool rgb_matrix_indicators_advanced_kb(uint8_t led_min, uint8_t led_max) {
+    if (!rgb_matrix_indicators_advanced_user(led_min, led_max)) {
+        return false;
+    }
+    if (pwron_flag == false) {
+        map_current_time = timer_read();
+        if (map_current_time < 3000) {
+            map_current_time++;
```

Comment:
Is this needed? 
Gets overwritten each run with
`map_current_time = timer_read();`

Only if timer_read() returns 2999 this lead to direct moving to the next state instead of the next call of the function.